### ISSUE TITLE
Feature Request: Add NavEnableKeyboard flag to the API

### ISSUE THREAD

**Issue by Aman-Anas** [pyimgui/pyimgui#338]
![image](https://github.com/pyimgui/pyimgui/assets/41600372/f85f12a8-e508-40a7-80cf-7decceb0b4be)

There is a configuration flag to enable keyboard controls, available in the imgui demo window. When activating via the test window, it works fine, however there doesn't seem to be a way to set this flag beforehand. Can this be added to the API? There are already some other config flags in there but it seems quite a few are missing.

**Comment by KinoxKlark** [comment]
Hello,

You have to set it through `IO`, it should look like

```python
io = imgui.get_io()
io.config_flags = io.config_flags | imgui.CONFIG_NAV_ENABLE_KEYBOARD
```

Possible config flags are:
```
imgui.CONFIG_NONE
imgui.CONFIG_NAV_ENABLE_KEYBOARD
imgui.CONFIG_NAV_ENABLE_GAMEPAD
imgui.CONFIG_NAV_ENABLE_SET_MOUSE_POS
imgui.CONFIG_NAV_NO_CAPTURE_KEYBOARD
imgui.CONFIG_NO_MOUSE
imgui.CONFIG_NO_MOUSE_CURSOR_CHANGE
imgui.CONFIG_IS_RGB
imgui.CONFIG_IS_TOUCH_SCREEN
```
which is the exhaustive set of config flags as far as I know.

Do you have any precise example in mind that I may have missed? (Note that we are mapping DearImGui v1.52, but I didn't see any more config flags in the latest release)

**Comment by KinoxKlark** [comment]
Yeah, you are right, it seems that they are not in the documentation.

This documentation definitely needs to be refactored. It is quite a mess ^^. In the mean time, if you don't find something, I can suggest you look for functions and flags directly in the `imgui/core.pyx`. The documentation is generated using the content of this file.

Thanks for letting me know!

**Comment by Aman-Anas** [comment]
Just a follow-up, how would you remove a config flag after you set it? I'm not familiar with how the `|` bitwise or operator works.

**Comment by Aman-Anas** [comment]
Ah, nevermind, the `^` operator seems to do the trick

```py
io = imgui.get_io()
io.config_flags |= imgui.CONFIG_NAV_ENABLE_KEYBOARD

# Remove flag
io.config_flags ^= imgui.CONFIG_NAV_ENABLE_KEYBOARD
```

**Comment by learn-more** [comment]
> Ah, nevermind, the `^` operator seems to do the trick
> 
> ```python
> io = imgui.get_io()
> io.config_flags |= imgui.CONFIG_NAV_ENABLE_KEYBOARD
> 
> # Remove flag
> io.config_flags ^= imgui.CONFIG_NAV_ENABLE_KEYBOARD
> ```

That toggles the flag, so it adds it if it was not there, or it removes it if it was there.
You are probably looking for:

```python
io.config_flags &= ~imgui.CONFIG_NAV_ENABLE_KEYBOARD
```
